spec/02-identifiers-names-and-scopes.md

```diff
@@ -17,9 +17,10 @@ which are collectively called _bindings_.
 Bindings of each kind are assigned a precedence which determines
 whether one binding can shadow another:
 
-1. Definitions and declarations that are local, inherited, or made
+1. Local definitions and declarations have the highest precedence.
```

Comment:
I noticed I was using local informally to mean "in lexical scope", i.e., in the current compilation unit. I assume originally it was intended to mean "local" as in not a member of the enclosing element. We say "top-level", cf ch 9. The dotty PR doesn't define "coincide". Oh, you just said "top-level". There is ambiguity about members of package objects (or I was confused at some point). I've always disliked the word "basic" in the title of ch 4. Closest for "what is local" in the normal sense is block statements 6.25 (not quite block exprs 6.11) where a def introduces a local name. There must be something about no overloading for locals, or is that by omission?

I did look at pursuing "coinciding" definitions yesterday, but I'm glad I gave up quickly.

Reply:
To recap, I didn't write "local definitions" to mean "not top-level" but to mean informally "local to this source file" or something. It reads well but is meaningless. The amendment is "D&D in lexical scope", but it must add "that are not top-level". Now I don't like the verbiage "made available by a package clause", but perhaps because of package objects, it can't just say "top-level definitions".

Reply:
oh the opening paragraph also says "local" to mean "lexical scope". I will delete that word as well.

Reply:
I made these edits and squashed before someone joked, "local yokel".